src/blend/src/Shared/Blend/Blend.lua

```diff
@@ -787,14 +787,17 @@ function Blend.Single(observable)
 				local copy = BrioUtils.clone(result)
 				maid._current = copy
 				sub:Fire(copy)
-				return copy
+			elseif result then
+				local current = Brio.new(result)
+				maid._current = current
+				sub:Fire(current)
+			else
+				maid._current = nil
 			end
-
-			local current = Brio.new(result)
-			maid._current = current
-			sub:Fire(current)
-
-			return current
+		end, function(...)
```

Comment:
we can pass sub:GetFailComplete() here. Note I think it's fine to complete this here, but a failure could cause other observables in the blend tree to also fail. Most of Nevermore doesn't really leverage the failure state that much.

Reply:
If you think it's safe to merge this, I'm down for it.

Reply:
I wanted the completion, I didn't realise this would could pass failures. I was only vaguely aware subscriptions had this - I know for certain I'm not handling it!

Though it's a breaking change, I would want to propagate this. I think hiding errors is bad. But silently failing is also bad, because I'm not handling this case and wouldn't know why my subscription stopped. I think subscriptions should print a warning (similar to promises) for an unhandled failure state, and then this should be merged?